### DISCUSSION TITLE
Flysky ANT protocol and AFHDS 3 protocol

### DISCUSSION THREAD

**Discussion by lamborbikini** [FrSkyRC/ETHOS-Feedback-Community#3247] [Q&A] [answered]
Flysky ANT protocol and AFHDS 3 protocol
are these two protocols supported? which protocol number should i choose?

Thank you!

**Comment by packetpilot** [comment, marked as answer]
It seems you're in the wrong place. [This](https://www.multi-module.org/using-the-module/protocol-details) may be the better well to drink from.

**Reply by lamborbikini** [reply]
Thank you very much!
